docs/_ug/commandSummary/itemCommands/edit.md

```diff
@@ -0,0 +1,2 @@
+<!-- markdownlint-disable-file first-line-h1 -->
+`edit id/INDEX_LIST [n/ITEM_NAME] [qty/QUANTITY] [unit/UNIT] [bgt/BOUGHT_DATE] [exp/EXPIRY_DATE]`
```

Comment:
Inconsistent labelling of `INDEX` field compared to other commands.

```suggestion
`edit INDEX [n/ITEM_NAME] [qty/QUANTITY] [unit/UNIT] [bgt/BOUGHT_DATE] [exp/EXPIRY_DATE]`
```

Reply:
Yes , we will use INDEX throughout. No longer using INDEX_LIST